test/unit/manipulation.js

```diff
@@ -2279,7 +2279,7 @@ QUnit.test( "Cloned, detached HTML5 elems (#10667,10670)", function( assert ) {
 	// First clone
 	$clone = $section.clone();
 
-	// This branch tests a known behaviour in modern browsers that should never fail.
+	// This branch tests a known behavior in modern browsers that should never fail.
```

Comment:
I'm okay with changing British to US spelling, but not particularly bothered about the original.